deepchem/trans/transformers.py

```diff
@@ -1318,6 +1323,23 @@ def gaussian_blur(self, sigma=0.2):
     """
     return scipy.ndimage.gaussian_filter(self.Image, sigma)
 
+  def crop(self, x_crop, y_crop):
+    """ Crops the image from the center.
+          Parameters:
+            x_crop - bound for x
+            y_crop - bound for y
+    """
+    y = self.Image.shape[0]
+    x = self.Image.shape[1]
+    x_start = x // 2 - (x_crop // 2)
+    y_start = y // 2 - (y_crop // 2)
+    return self.Image[y_start:y_start + y_crop, x_start:x_start + x_crop]
+
+  def convert2gray(self):
+    """ Converts the image to grayscale
+    """
+    return np.dot(self.Image[..., :3], [0.2989, 0.5870, 0.1140])
```

Comment:
Could you add a reference for the magic numbers here? (0.2989 etc)

Reply:
Yep, I've borrowed it from the MATLAB implementation of the rgb2gray function.Have included a link too.

Reply:
Those coefficients correspond to the Y' component of the [Y'UV color system](https://en.wikipedia.org/wiki/YUV).  They were chosen to reflect how the human visual system perceives "brightness".  It's dominated by the amount of green light, and the amount of blue light has only a very small effect on how bright something appears.